components/kyma-environment-broker/internal/orchestration/handlers/kyma_handler.go

```diff
@@ -50,13 +58,23 @@ func (h *kymaHandler) createOrchestration(w http.ResponseWriter, r *http.Request
 			return
 		}
 	}
+
+	// validate target
 	err := h.validateTarget(params.Targets)
 	if err != nil {
 		h.log.Errorf("while validating target: %v", err)
 		httputil.WriteErrorResponse(w, http.StatusBadRequest, errors.Wrapf(err, "while validating target"))
 		return
 	}
 
+	// validate Kyma version
+	err = h.ValidateKymaVersion(params.Version)
```

Comment:
can we just omit this check if version wasn't provided?

Reply:
In my opinion, not quite. We should have log details if possible when the validation fails.